src/strongbus_spec.ts

```diff
@@ -786,15 +786,61 @@ describe('Strongbus.Bus', () => {
       sub2();
     });
 
-    it('raises "error" when errors are thrown in the listener', () => {
-      const error = new Error('Error in callback');
-      bus.on('bar', () => {
-        throw error;
+    describe('error events', () => {
```

Comment:
All these new specs are just capturing existing behavior. They all pass against the `master` branch.